packages/server/graphql/public/types/Company.ts

```diff
@@ -174,43 +202,7 @@ const Company: CompanyResolvers = {
   },
 
   organizations: async ({id: domain}, _args, {authToken, dataLoader}) => {
-    const organizations = await dataLoader.get('organizationsByActiveDomain').load(domain)
-    // only superusers can see across ALL organizations.
-    // users can only see the organizations they are apart of
-    if (isSuperUser(authToken)) return organizations
-    const orgIds = organizations.map(({id}) => id)
-    const viewerId = getUserId(authToken)
-    const allOrganizationUsers = (
-      await Promise.all(
-        orgIds.map((orgId) => {
-          return dataLoader.get('organizationUsersByUserIdOrgId').load({orgId, userId: viewerId})
-        })
-      )
-    ).filter(isValid)
-    // If suggestedTier === enterprise, that means the user is allowed to see across
-    // all organizations, even the ones they are not a member of!
-    const isViewerAllowedToSeeAll = allOrganizationUsers.some(
-      ({suggestedTier}) => suggestedTier === 'enterprise'
-    )
-    if (isViewerAllowedToSeeAll) return organizations
-    // Pro-qualified or unqualified users can only see orgs that they are apart of
-    const allowedOrgIds = allOrganizationUsers.map(({orgId}) => orgId)
-    return organizations.filter((organization) => allowedOrgIds.includes(organization.id))
-  },
-
-  viewerOrganizations: async ({id: domain}, _args, {authToken, dataLoader}) => {
```

Comment:
I removed `viewerOrganizations` and removed `if (isSuperUser(authToken)) return organizations` check from `organizations`

`viewerOrganizations` seems to be a "workaround" to avoid this check. But do we need this check at all? It feels confusing as it breaks expected logic for super users.

If we need this later I think its better to create some admin tools or `allDomainOrganizations` prop